src/rocm_docs/theme.py

```diff
@@ -89,11 +90,22 @@ def _update_theme_options(app: Sphinx) -> None:
             0, "components/left-side-menu"
         )
 
+    header_latest_version = requests.get(
+        "https://raw.githubusercontent.com/RadeonOpenCompute/rocm-docs-core/header-versions/latest_version.txt"
+    ).text
+    header_release_candidate = requests.get(
+        "https://raw.githubusercontent.com/RadeonOpenCompute/rocm-docs-core/header-versions/release_candidate.txt"
+    ).text
+
     default_config_opts = {
         "html_show_sphinx": False,
         "html_favicon": "https://www.amd.com/themes/custom/amd/favicon.ico",
         "notfound_context": {"title": "404 - Page Not Found"},
         "notfound_template": "404.html",
+        "html_context": {
+            "header_latest_version": header_latest_version,
+            "header_release_candidate": header_release_candidate,
+        },
```

Comment:
This should provide the variables needed by the jinja header template, but the RTD build still seems to fail for some reason

```
  File "/home/docs/checkouts/readthedocs.org/user_builds/advanced-micro-devices-rocm-docs-core/envs/500/lib/python3.8/site-packages/rocm_docs/rocm_docs_theme/flavors/rocm/header.jinja", line 4, in template
    {% elif release_candidate_version in branch %}
TypeError: 'in <string>' requires string as left operand, not Undefined
```